### ISSUE TITLE
Figure 1

### ISSUE THREAD

**Issue by lvmengxi** [lvmengxi/2017-Oct-DataIncubator#1]
![image](https://user-images.githubusercontent.com/12598092/32295404-8ea9b810-bf1f-11e7-872e-102943bdfd94.png)

![2017-10-30_113920_pic1](https://user-images.githubusercontent.com/12598092/32294936-2248b5f0-bf1e-11e7-913f-8ba86bdf31ed.png)

Figure 1. Customers of genetic variance testing present difference attitude towards their privacy protection. (A) Among 2812 customers, 18% of them published their gender information, of which 15% are male and 3% are female. (B) By analyzing their genetic test report, I found that 1632 (58%) customers are female and 1183 (42%) are male. In addition, people aged 29-39 years old are more likely to reveal their genetic test results and personal information. Distribution of age for (C) female customers, (D) male customers, (E) overall customers, and (F) customers without gender information.  
**To summarize, female customers of genetic variance test service more tend to protect their gender information, but not their age information when compared to male customers.**
